### PR TITLE
Add documentation for Aquacell integration

### PR DESCRIPTION
**Pull request by Jordi1990**
## Proposed change

Adds documentation for the Aquacell integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#117117
- Link to parent pull request in the Brands repository: home-assistant/brands#5445
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/aquacell.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ha_config_flow: true
+ha_codeowners:
+  - '@Jordi1990'
+ha_domain: aquacell
+ha_platforms:
+  - sensor
+ha_integration_type: device
+---
+
+AquaCell is a water softening device. The **AquaCell** integration allows you to monitor your AquaCell device in Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/aquacell.markdown]:
```suggestion
AquaCell is a water-softening device. The **AquaCell** {% term integration %} allows you to monitor your AquaCell device in Home Assistant.
```
```suggestion
AquaCell is a water softening device. The **AquaCell** integration allows you to monitor your AquaCell device in Home Assistant.
```

File: source/_integrations/aquacell.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+This integration provides {% term sensors %} for the following information from the AquaCell device:
+
+- Percentage of salt remaining.
+- Number of days until 0% salt level is reached.
+- Firmware version.
+- Device name.
+- iLid battery level.
+- Last update time.
+- WiFi signal level.
```
Comment (c0ffeeca7) [inline source/_integrations/aquacell.markdown]:
```suggestion
- Wi-Fi signal level.
```
```suggestion
- WiFi signal level.
```

File: source/_integrations/aquacell.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - Sensor
+ha_release: 2024.6
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@Jordi1990'
+ha_domain: aquacell
+ha_platforms:
+  - sensor
+ha_integration_type: device
```
Comment (c0ffeeca7) [inline source/_integrations/aquacell.markdown]:
```suggestion
ha_integration_type: integration
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @Jordi1990 :+1:

**Comment by Jordi1990** [comment]
Thanks @c0ffeeca7
Made some final small changes to match with the changed code.

File: source/_integrations/aquacell.markdown [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+</div>
+
+## Sensors
+
+This integration provides {% term sensors %} for the following information from the AquaCell device:
+
+- Percentage of salt remaining.
+- Time remaining until 0% salt level is reached.
+- iLid battery level.
+- Wi-Fi signal level.
```
Comment (c0ffeeca7) [inline source/_integrations/aquacell.markdown]:
```suggestion
- Wi-Fi signal strength.
```

**Review by c0ffeeca7 — APPROVED** [review]

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @Jordi1990 :+1:

File: source/_integrations/aquacell.markdown [hunk 5]
```diff
@@ -0,0 +1,37 @@
+---
+title: AquaCell
+description: Instructions on how to integrate AquaCell with Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 2024.6
```
Comment (joostlek) [inline source/_integrations/aquacell.markdown]:
```suggestion
ha_release: 2024.7
```

**Review by joostlek — APPROVED** [review]

**Pull request merged**